.github/workflows/release.yml

```diff
@@ -23,29 +28,86 @@ jobs:
         run: |
           make build
           chmod -R +x dist
-          zip -r release.zip dist
+      - name: Make checksums
+        run: make checksums
+      - name: store artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: binaries
+          path: dist
 
-      - name: Release binaries
-        uses: softprops/action-gh-release@v1
-        if: startsWith(github.ref, 'refs/tags/')
+  build-push-linux-multi:
+    name: Build & push linux/amd64 and linux/arm64
+    needs: [ build-binaries ]
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [ argo-events ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
         with:
-          files: release.zip
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          platforms: amd64,arm64
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
-      - name: Docker Login
-        uses: Azure/docker-login@v1
+      - name: Download binaries
+        uses: actions/download-artifact@v2
         with:
-          login-server: quay.io
+          name: binaries
+          path: dist/
+
+      - name: Registry Login
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
           username: ${{ secrets.QUAYIO_USERNAME }}
           password: ${{ secrets.QUAYIO_PASSWORD }}
 
-      - name: Build & Push Linux Docker Images
+      - name: set tag csv value
+        id: tag
         env:
           QUAYIO_ORG: ${{ secrets.QUAYIO_ORG }}
+          TARGET: ${{ matrix.target }}
         run: |
           tag=$(basename $GITHUB_REF)
           if [ $tag = "master" ]; then
             tag="latest"
           fi
-          make image IMAGE_TAG=${tag} IMAGE_NAMESPACE=${QUAYIO_ORG} DOCKER_PUSH=true
+          image_name="${QUAYIO_ORG}/${TARGET}:${tag}"
+          echo "::set-output name=TAG_LIST::$image_name"
+
+      - name: Container build and push
```

Comment:
Can we utilize `make image-multi` command to build and push images?

Reply:
it would not be efficient as is.
As we want to keep the container build short to avoid any quay or docker registry login timeout, we build the binaries in the previous step. it also allows to attach the binaries to the release.
But in the make image-multi we actually build the binaries as part of it.

What I could do is to have a switch in the make image-multi to optionally not build the binaries. I send it in few min

Reply:
done